### PR TITLE
Preparing washboard for MVP

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Fixes https://github.com/wasmCloud/wasmCloud/issues/216, Related to #68 

Completed MVP checkmarks in the above wasmCloud#216 issue

### CODE REVIEW

File: wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex [hunk 1]
```diff
@@ -12,32 +12,61 @@ defmodule DefineLinkComponent do
...
-          "actor_id" => actor_id,
-          "contract_id" => contract_id,
-          "link_name" => link_name,
-          "provider_id" => provider_id,
-          "values" => values
-        },
+        linkdef,
         socket
       ) do
+    IO.inspect(linkdef)
```
Comment (autodidaddict) [inline wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex:18]: Can probably get rid of this ;)

File: wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex [hunk 2]
```diff
@@ -12,32 +12,61 @@ defmodule DefineLinkComponent do
...
+
     values_map =
-      values
-      |> String.split(",")
-      |> Enum.flat_map(fn s -> String.split(s, "=") end)
-      |> Enum.chunk_every(2)
-      |> Enum.map(fn [a, b] -> {a, b} end)
-      |> Map.new()
+      case values do
+        "" ->
```
Comment (autodidaddict) [inline wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex:29]: does `"" | nil ->` work?

**Review by autodidaddict — CHANGES_REQUESTED** [review]
LGTM, just remove 1 line of debug

File: wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex (hunk 2 above)
Comment (brooksmtownsend) [reply wasmcloud_host/lib/wasmcloud_host_web/live/components/define_link_component.ex:29]: It does not, I tried `||` as well and elixir doesn't seem to allow the conditionals in the pattern matching statement. Elixir also interprets `|` as a list operation 😢

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**